https://github.com/PyFilesystem/pyfilesystem2/pull/587.patch

From 2d2acf6cba038eb7e0a1df0d702832f0cac464a3 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz93@gmail.com>
Date: Sun, 15 Sep 2024 16:53:20 -0400
Subject: [PATCH] tests: fix deprecated and removed aliases to assertEqual

This file already uses the "correct" name a bunch of times. Both names
have been around for the same amount of time but the version with an "s"
at the end was deprecated in 3.2 and removed in 3.12.
---
 tests/test_move.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git tests/test_move.py tests/test_move.py
index 8eb1af75..7dd5b114 100644
--- tests/test_move.py
+++ tests/test_move.py
@@ -167,7 +167,7 @@ def test_move_file_overwrite(self, _, fs_url):
             self.assertFalse(src.exists("target.txt"))
             self.assertFalse(dst.exists("file.txt"))
             self.assertTrue(dst.exists("target.txt"))
-            self.assertEquals(dst.readtext("target.txt"), "source content")
+            self.assertEqual(dst.readtext("target.txt"), "source content")
 
     @parameterized.expand([("temp", "temp://"), ("mem", "mem://")])
     def test_move_file_overwrite_itself(self, _, fs_url):
@@ -177,7 +177,7 @@ def test_move_file_overwrite_itself(self, _, fs_url):
             tmp.writetext("file.txt", "content")
             fs.move.move_file(tmp, "file.txt", tmp, "file.txt")
             self.assertTrue(tmp.exists("file.txt"))
-            self.assertEquals(tmp.readtext("file.txt"), "content")
+            self.assertEqual(tmp.readtext("file.txt"), "content")
 
     @parameterized.expand([("temp", "temp://"), ("mem", "mem://")])
     def test_move_file_overwrite_itself_relpath(self, _, fs_url):
@@ -188,7 +188,7 @@ def test_move_file_overwrite_itself_relpath(self, _, fs_url):
             new_dir.writetext("file.txt", "content")
             fs.move.move_file(tmp, "dir/../dir/file.txt", tmp, "dir/file.txt")
             self.assertTrue(tmp.exists("dir/file.txt"))
-            self.assertEquals(tmp.readtext("dir/file.txt"), "content")
+            self.assertEqual(tmp.readtext("dir/file.txt"), "content")
 
     @parameterized.expand([(True,), (False,)])
     def test_move_file_cleanup_on_error(self, cleanup):
